packages/cli/src/services/__tests__/linter.test.ts

```diff
@@ -212,7 +212,9 @@ describe('Linter service', () => {
 
     describe('when single ruleset option provided', () => {
       it('outputs "does not exist" error', () => {
-        return expect(run(`lint ${validOas3SpecPath} -r non-existent-path`)).rejects.toThrow('Cannot find module');
+        return expect(run(`lint ${validOas3SpecPath} -r non-existent-path`)).rejects.toThrow(
```

Comment:
Might this error message be clearer if it was along the lines of `Could not read ruleset at {path}`? What's the `resolve entry module` lingo in here?

Reply:
It's the message Rollup.js prints.

Reply:
Is there a way to provide a better message - seems like a common use case re missing ruleset at the path given? If it's a pain then let's skip, not critical